### PR TITLE
Update `Contest points` and `Official beatmapping contest support`

### PR DESCRIPTION
**Pull request by Hiviexd**
changes were discussed and approved by the contest committee

### CODE REVIEW

File: wiki/Contests/Contest_points/en.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 Contest points are points that are used as a reward in official mapping contests. They are mainly used to determine the eligibility of a user to receive the *Elite Mapper* title, which is awarded to users who have reached **6 points**.
 
+Contest points can also be granted as a benefit of [official beatmapping contest support](/wiki/Contests/Official_support) under [certain circumstances](/wiki/Contests/Official_support#eligibility).
+
 ## Points key
 
-For solo contests, contest points are based on how many valid submissions are created.
+For solo contests, contest points are based on how many valid submissions are created. These are the general guidelines for point distribution, and exceptions may be granted on a case-by-case basis with the discretion of the [Contest Committee](/wiki/People/Tournament_Committee#contest-committee) when requesting official beatmapping contest support.
```
Comment (MChecaH) [inline wiki/Contests/Contest_points/en.md:9]:
```suggestion
For solo contests, contest points are based on how many valid submissions are created. These are the general guidelines for point distribution, and exceptions may be granted on a case-by-case basis at the discretion of the [Contest Committee](/wiki/People/Tournament_Committee#contest-committee) when requesting official beatmapping contest support.
```

File: wiki/Contests/Official_support/en.md [hunk 2]
```diff
@@ -68,3 +69,7 @@ If approved, you will need to submit the following:
 - An appropriately designed badge (see profile badge design criteria on the [official tournament support article](/wiki/Tournaments/Official_support#profile-badges)).
 
 In exceptionally rare cases, a contest may be granted the privilege of awarding badges to its top three finalists. A contest that has received this distinction is recognised for attaining the highest level of production quality and organisation possible — this is given exclusively at the discretion of the osu! team. Hosts will be contacted directly if applicable.
+
+## Notes
+
+- [^exception-note]: Exceptions may be granted on a case-by-case basis with the discretion of the [Contest Committee](/wiki/People/Tournament_Committee#contest-committee).
```
Comment (MChecaH) [inline wiki/Contests/Official_support/en.md:75]:
```suggestion
- [^exception-note]: Exceptions may be granted on a case-by-case basis at the discretion of the [Contest Committee](/wiki/People/Tournament_Committee#contest-committee).
```

**Review by MChecaH — CHANGES_REQUESTED** [review]
Change to a more appropriate preposition.

**Review by MChecaH — APPROVED** [review]

File: wiki/Contests/Official_support/en.md [hunk 3]
```diff
@@ -68,3 +69,7 @@ If approved, you will need to submit the following:
 - An appropriately designed badge (see profile badge design criteria on the [official tournament support article](/wiki/Tournaments/Official_support#profile-badges)).
 
 In exceptionally rare cases, a contest may be granted the privilege of awarding badges to its top three finalists. A contest that has received this distinction is recognised for attaining the highest level of production quality and organisation possible — this is given exclusively at the discretion of the osu! team. Hosts will be contacted directly if applicable.
+
+## Notes
+
+- [^exception-note]: Exceptions may be granted on a case-by-case basis at the discretion of the [Contest Committee](/wiki/People/Tournament_Committee#contest-committee).
```
Comment (Walavouchey) [inline wiki/Contests/Official_support/en.md:75]: incorrect formatting

```suggestion
[^exception-note]: Exceptions may be granted on a case-by-case basis at the discretion of the [Contest Committee](/wiki/People/Tournament_Committee#contest-committee).
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**